### ISSUE TITLE
"Replace Glow Squid Renderer" config isn't working in 1.18.2

### ISSUE THREAD

**Issue by Epicava32** [minecraftabnormals/Upgrade-Aquatic#370]
I wanted to create randomized textures for the glow squid but this feature overwrites it and when I try to disable it, it isn't working.

**Comment by Dirt-Eater** [comment]
I also have been having a similar issue, I am trying to use the squid textures from Thorolhugil and this mod overwrites the glowing squid textures. I attempted to delete anything that mentions glowing squids in the mod files hoping that would solve it but that just causes it to crash. Replacing the glowing squid texture files in upgrade aquatic with the ones from Thorolhugil results in purple and black missing textures. Hoping there is a way to use this mod without the changes to glowing squids as I love all the other features but do not like the vanilla squid design and would strongly prefer to be able to use the resource pack I found.
